src/utils/registry.ts

```diff
@@ -0,0 +1,24 @@
+import { NetworkNames } from '../constants';
+import { detectProvider, MarinaProvider } from 'marina-provider';
+
+const Registries = {
+  [NetworkNames.MAINNET]: 'https://raw.githubusercontent.com/TDex-network/tdex-registry/master/registry.json',
+  [NetworkNames.TESTNET]: 'https://joaobordalo.com/labs/tdex-registry/registry.json', // TODO: change this
+}
+
+/**
+ * Get tdex registry url based on network selected on Marina
+ * @returns url
+ */
+export async function getRegistryURL(): Promise<string> {
```

Comment:
also here, seems more clean to decouble from Svelte store, and having it passed down as argument

Reply:
Same issue, is used inside a module imported by the Svelte component, not sure on how to pass params from the Svelte component to the module when importing.

Reply:
passing down to `fetchProvidersAndMarketsAndSet` should solve also here

Reply:
Commit  99635d4f should fix this.